examples/single_particle/demo_FLR.jl

```diff
@@ -0,0 +1,56 @@
+# finite Larmor radius effect

```

Comment:
Start with a capital letter. Same for other files.

Reply:
> Start with a capital letter. Same for other files.

Is there anywhere a list of rules related to informal text like code comments? I would like to know about it.